<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kholloscope</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="info">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_File_Create"/>
    <addaction name="action_File_Open"/>
    <addaction name="separator"/>
    <addaction name="action_File_Quit"/>
   </widget>
   <widget class="QMenu" name="menuBDD">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;BDD</string>
    </property>
    <addaction name="action_DB_Students"/>
    <addaction name="action_DB_Groups"/>
    <addaction name="action_DB_Subjects"/>
    <addaction name="action_DB_Kholleurs"/>
   </widget>
   <widget class="QMenu" name="menuPlanning">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Planning</string>
    </property>
    <addaction name="action_Schedule_Timetable"/>
    <addaction name="action_Schedule_Students_Groups"/>
    <addaction name="action_Schedule_Kholles"/>
    <addaction name="action_Schedule_Events"/>
   </widget>
   <widget class="QMenu" name="menuKholles">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Kholles</string>
    </property>
    <addaction name="action_Kholles_Interface"/>
    <addaction name="action_Kholles_Generate"/>
    <addaction name="action_Kholles_LastChanges"/>
    <addaction name="action_Kholles_Historic"/>
   </widget>
   <widget class="QMenu" name="menu_Aide">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_AboutIt"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuBDD"/>
   <addaction name="menuPlanning"/>
   <addaction name="menuKholles"/>
   <addaction name="menu_Aide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_File_Open">
   <property name="text">
    <string>&amp;Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_File_Create">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C&amp;réer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_File_Delete">
   <property name="text">
    <string>&amp;Supprimer</string>
   </property>
  </action>
  <action name="action_File_Configuate">
   <property name="text">
    <string>&amp;Configurer</string>
   </property>
  </action>
  <action name="action_File_Quit">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_DB_Students">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Elèves</string>
   </property>
  </action>
  <action name="action_DB_Groups">
   <property name="text">
    <string>&amp;Groupes</string>
   </property>
  </action>
  <action name="action_DB_Subjects">
   <property name="text">
    <string>&amp;Matières</string>
   </property>
  </action>
  <action name="action_DB_Kholleurs">
   <property name="text">
    <string>E&amp;nseignants</string>
   </property>
  </action>
  <action name="action_Schedule_Timetable">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Emploi du temps</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Schedule_Students_Groups">
   <property name="text">
    <string>&amp;Lien élèves-groupes</string>
   </property>
  </action>
  <action name="action_Schedule_Kholles">
   <property name="text">
    <string>&amp;Horaires de kholles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_Schedule_Events">
   <property name="text">
    <string>Evènements &amp;spéciaux</string>
   </property>
  </action>
  <action name="action_Kholles_Interface">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Interface</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Kholles_Generate">
   <property name="text">
    <string>&amp;Générer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Kholles_Historic">
   <property name="text">
    <string>&amp;Historique</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Contacts</string>
   </property>
  </action>
  <action name="action_AboutIt">
   <property name="text">
    <string>A p&amp;ropos de</string>
   </property>
  </action>
  <action name="action_Kholles_LastChanges">
   <property name="text">
    <string>&amp;Outil de dépannage</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
